### PR TITLE
Circular Linked List In C

### PR DESCRIPTION
**Pull request by Mansiuniyal**
## 🛠️ Issue (Number)

Issue no #265 

## 👨‍💻 Changes proposed

Added the program of Circular linked list in C

## ✔️ Check List (Check all the applicable boxes) 

- [✔️] My code follows the code style of this project.
- [✔️] This PR does not contain plagiarized content.
- [✔️] The title of my pull request is a short description of the requested changes.

## 📷 Screenshots
Screenshot (1005)

### CODE REVIEW

File: C/Sorting/Shell_sort.c [hunk 1]
```diff
@@ -0,0 +1,37 @@
+#include<stdio.h>
+#include<stdlib.h>
+
```
Comment (zugzwang03) [inline C/Sorting/Shell_sort.c:3]: explain approach here

**Review by 072arushi — APPROVED** [review]
Approved

File: C/Sorting/Shell_sort.c (hunk 1 above)
Comment (Mansiuniyal) [reply C/Sorting/Shell_sort.c:3]: @zugzwang03 The code you mentioned is of shell sort for which I have created separate PR and have done the modifications there but that PR is not merged till now. This issue is of Circular Linked list for which I have added the code.

Comment (zugzwang03) [reply C/Sorting/Shell_sort.c:3]: creating branches would have been better though

**Review by Rishikavishnoi — APPROVED** [review]

**Review by Kumar-laxmi — CHANGES_REQUESTED** [review]
Please add the issue number of **Shell Sort**

**Comment by Kumar-laxmi** [comment]
@Mansiuniyal Why are you adding shell sort in this PR when you have opened PR #276

**Pull request closed without merging**